### ISSUE TITLE
Linting of multiple instance of COPY to same location

### ISSUE THREAD

**Issue by Shiva7071007** [hadolint/hadolint#439]
- [] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
Show linting error while doing to avoid creating unnecessary layers
COPY ./file1 ./temp
COPY ./file2  ./temp
COPY ./file3 ./temp

### Actual behavior
No linting error found 

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
(paste your output here)
```

**Dockerfile** (if relevant)

```Dockerfile
(paste content of your Dockerfile)
```

**Additional environment details (OS, stack version, etc.)**
